sunpy/net/solarnet/solarnet.py

```diff
@@ -0,0 +1,161 @@
+             /service/api/svo/metadata_xrt/20061123130943928/ 20061123130943928 2006-11-23T13:09:43.928Z 2006-11-23T13:09:44.101Z    0.88 ... v2014-Oct-20 830.976806641 8.22879981995 -162.474563599 8.22879981995  282240
+             /service/api/svo/metadata_xrt/20061123131040622/ 20061123131040622 2006-11-23T13:10:40.622Z 2006-11-23T13:10:40.927Z    0.88 ... v2014-Oct-20 831.645935059 2.05719995499 -157.406402588 2.05719995499 4213440
+    <BLANKLINE>
+    <BLANKLINE>
+    """
+
+    @property
+    def info_url(self):
+        return 'https://solarnet2.oma.be'
+
+    def search(self, *query):
+        """
+        Searches the SVO for datasets matching the query.
+
+        Parameters
+        ----------
+        *query : `~sunpy.net.attrs`
+            Attributes to search for datasets.
+
+        Returns
+        -------
+        `~sunpy.net.base_client.QueryResponseTable`
+            A table containing the search results.
+
+        Examples
+        --------
+        >>> from sunpy.net import Fido, attrs as a
+        >>> query = [a.solarnet.Dataset.swap_level_1 , a.solarnet.Limit(2), a.solarnet.Tags.moon_transit]
+        >>> search_results = Fido.search(*query)   # doctest: +REMOTE_DATA
+        >>> search_results  # doctest: +REMOTE_DATA
+        <sunpy.net.fido_factory.UnifiedResponse object at ...>
+        Results from 1 Provider:
+        <BLANKLINE>
+        2 Results from the SOLARNETClient:
+        Source: https://solarnet2.oma.be
+        <BLANKLINE>
+            Tags                      Resource_uri                           Oid               Date_beg                 Date_end         ...          Ttemp2         Wavelnth          Wcsname           F_size
+            ---- ------------------------------------------------------ -------------- ------------------------ ------------------------ ... ----------------------- -------- ------------------------- -------
+                 /service/api/svo/metadata_swap_level_1/20091120082529/ 20091120082529 2009-11-20T08:25:29.027Z 2009-11-20T08:25:30.027Z ... 2009-11-20T08:28:58.000      174 Helioprojective-cartesian 2113920
+                 /service/api/svo/metadata_swap_level_1/20091120082542/ 20091120082542 2009-11-20T08:25:42.027Z 2009-11-20T08:25:44.027Z ... 2009-11-20T08:28:58.000      174 Helioprojective-cartesian 2113920
+        <BLANKLINE>
+        <BLANKLINE>
+        """
+        results = []
+        query = and_(*query)
+        block = walker.create(query)[0]
+
+        if "datasets" in block:
+            url = _BASE_URL.format(block["datasets"])
+            block.pop("datasets")
+
+        req = requests.get(url, params=block)
+        data = req.json()["objects"]
+
+        for i in data:
+            # Exclude fits_header and data_location to avoid clumsiness
+            filtered_data = {key.capitalize(): value for key, value in i.items() if key not in ["fits_header", "data_location"]}
```

Comment:
Rather than doing this and not putting them in the table at all, I think the best thing to do would be to add them to `hide_keys` so that they aren't shown unless the user wants to see them.